pygmt/src/grd2xyz.py

```diff
@@ -0,0 +1,78 @@
+"""
+grd2xyz - Convert grid to data table
+"""
+import numpy as np
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(R="sequence")
+def grd2xyz(grid, xyz_format="a", **kwargs):
+    r"""
+    Create xyz tables from grid files.
+
+    Read a binary 2-D grid files and write out
+    xyz-triplets in ASCII [or binary] format to a standard output.
+
+    Full option list at :gmt-docs:`grd2xyz.html`
+
+    {aliases}
+
+    Parameters
+    ----------
+    grid : str or xarray.DataArray
+        The file name of the input grid or the grid loaded as a DataArray.
+        This is the only required parameter.
+    xyz_format : str
```

Comment:
```suggestion
    output_type : str
```
I would prefer output_type or data_format so that the parameter name can be consistent across all functions that use the future return_table helper